frontend/packages/topology/integration-tests/features/topology/topology-chart-area-visual.feature

```diff
@@ -273,3 +273,22 @@ Feature: Topology chart area
               And user sees shortcut for Access create connector handle
               And user sees shortcut for Qpen quick search modal
               And user sees shortcut for Drag and drop a JAR file into Topology
+
+
+        @regression @manual @5722
```

Comment:
We should be consistent with the tags. For everything that's not yet automated, do we use the tag @manual, or @to-do? Similarly @ODC-5722 vs @5722. I've seen both looking on other PR's

Reply:
If it's a manual scenario, then we are adding @manual 
@to-do, If the scenario is able to automate, but not yet automated. then we will add this tag
@sanketpathak  - Please update the epic number as @ODC-5722

Reply:
Updated